app/src/main/res/values/themes.xml

```diff
@@ -14,4 +14,10 @@
         <item name="android:windowLightStatusBar">true</item>
         <!-- Customize your theme here. -->
     </style>
+
+    <style name="Theme.App.Starting" parent="Theme.SplashScreen">
```

Comment:
I would suggest change name of this Theme to something `Theme.App.AkademiaAndroida.withSplash`